protocol/messenger_handler.go

```diff
@@ -583,6 +583,8 @@ func (m *Messenger) handleSyncChats(messageState *ReceivedMessageState, chats []
 func (m *Messenger) HandleSyncInstallationContactV2(state *ReceivedMessageState, message *protobuf.SyncInstallationContactV2, statusMessage *v1protocol.StatusMessage) error {
 	// Ignore own contact installation
```

Comment:
This comment is no longer on the right line

Reply:
Sorry, was testing something else and that left. Removed now.